scripts/validate-release

```diff
@@ -28,7 +28,11 @@ function parse_tag() {
 }
 
 function check_release_branch() {
-    TAG_BRANCH=$(git branch --all -q --contains $GIT_TAG | grep -v origin$ | grep -v "HEAD" | sed -e 's/^[[:space:]]*//')
+    TAG_BRANCH=$(git branch --all -q --contains $GIT_TAG | grep origin | grep -v origin$ | grep -v "HEAD" | sed -e 's/^[[:space:]]*//')
+    if [ "$TAG_BRANCH" == "remotes/origin/master" ] && [ "$MAJOR.$MINOR" == "1.22"  ]; then
```

Comment:
Hmm, I wonder if we can avoid hardcoding the Kubernetes version that the master branch is on. Maybe if we're on master then we grab the version out of go.mod and check to see if that matches the tag?

`grep 'k8s.io/kubernetes v' go.mod | head -n1 | awk '{print $2}'`